openedx/features/learner_profile/__init__.py

```diff
@@ -8,12 +8,16 @@
 # Namespace for learner profile waffle flags.
 WAFFLE_FLAG_NAMESPACE = WaffleFlagNamespace(name='learner_profile')
 
-# Waffle flag for showing a message about the new profile features.
-# TODO: LEARNER-2554: 09/2017: Remove flag once message is no longer needed.
-SHOW_PROFILE_MESSAGE = WaffleFlag(WAFFLE_FLAG_NAMESPACE, 'show_message')
```

Comment:
**Note to reviewers:** The `SHOW_PROFILE_MESSAGE` flag (created in 09/2017) is no longer needed so I chose to remove it rather than document it at this point.